### ISSUE TITLE
TypeError during prove payment, silent fail.

### ISSUE THREAD

**Issue by qubenix** [monero-project/monero-gui#1123]
When trying to prove payment there is a TypeError that is displayed in the term log, but the gui fails silently. Here is the output of the term:

```
qml: getProof: Generate clicked: txid 74baa4cbfe47ea6656c25a0a91b6b0124f1c448f8424a950b68bf3955cc94d44, address 9vz3MBqiWwLbsF4K9WCSgpH3UsW4pknan2xmyGgsCBfgA1pD8jC3XpW3s4mVffyErZfWuC2s4HqZ1JJpBXB1FgNwPCHrDm3, message: 
qml: Getting payment proof: 
qml: 	txid:  74baa4cbfe47ea6656c25a0a91b6b0124f1c448f8424a950b68bf3955cc94d44 , address:  9vz3MBqiWwLbsF4K9WCSgpH3UsW4pknan2xmyGgsCBfgA1pD8jC3XpW3s4mVffyErZfWuC2s4HqZ1JJpBXB1FgNwPCHrDm3 , message:  
qrc:///main.qml:785: TypeError: Property 'startsWith' of object InProofV11XFBohNW5t1BgV64L3MKepTb84xMT8zxWguBUUnHtkLLAx3sXbp1U11bDuHoGJYoh7542NP1Rdo54AMMWmYrQa4qT3ZrLfbftEKbyVN5qPj2dNQuaA3AfUFdGi9wj2pKixPk is not a function
```

**Comment by stoffu** [comment]
Hmm, I wrote that code but I don't observe that behavior (macOS 10.12.6). Is there anyone else with this error?

**Comment by qubenix** [comment]
Sorry, I should have mentioned that I'm on Debian 9.

**Comment by stoffu** [comment]
OK, I also observe this issue on Ubuntu 16.04.

**Comment by stoffu** [comment]
#1124

**Comment by qubenix** [comment]
Closed #1123 via 5db91854091e90c819ef06d9ba1d81998faff350
